src/components/Process/Header.tsx

```diff
@@ -229,6 +233,60 @@ const ProcessHeader = () => {
   )
 }
 
+// todo(kon): disable this cast when ready on the sdk
+type Census3StrategyTokenIcon = Census3StrategyToken & { iconURI: string }
+
+const GitcoinStrategyInfo = () => {
+  const { t } = useTranslation()
+  const { election } = useElection()
+
+  if (!election || (election && !election?.meta?.strategy)) return ''
+  const strategy: Strategy = election.get('strategy')
+
+  const score = strategy.tokens['GPS'].minBalance
+  const firstParenthesesMatch = strategy.predicate.match(/\(([^)]+)\)/)
+  let unionTypeString: string | null = null
+  if (firstParenthesesMatch) {
+    const unionType = firstParenthesesMatch[1].split(' ')[1] // split by space and get the second element which should be the union type
```

Comment:
```suggestion
    const [,unionType] = firstParenthesesMatch[1].split(' ') // split by space and get the second element which should be the union type
```

Reply:
BTW... the comment saying "should be the union type" makes me thing this code block could fail, so it should be inside a try/catch.